src/pkg/packager/dev.go

```diff
@@ -105,3 +107,30 @@ func (p *Packager) DevDeploy(ctx context.Context) error {
 	// cd back
 	return os.Chdir(cwd)
 }
+
+// Lint ensures a package is valid & follows suggested conventions
+func (p *Packager) Lint(ctx context.Context) (err error) {
+	config.CommonOptions.Confirm = true
+
+	if err := os.Chdir(p.cfg.CreateOpts.BaseDir); err != nil {
+		return fmt.Errorf("unable to access directory %q: %w", p.cfg.CreateOpts.BaseDir, err)
+	}
+
+	lintFindings, err := lint.Validate(ctx, p.cfg.CreateOpts)
+	if err != nil {
+		return fmt.Errorf("linting failed: %w", err)
+	}
+
+	if len(lintFindings) == 0 {
+		message.Successf("0 findings for %q", p.cfg.Pkg.Metadata.Name)
+		return nil
+	}
+
+	lint.PrintFindings(lintFindings, types.SevWarn, p.cfg.CreateOpts.BaseDir, p.cfg.Pkg.Metadata.Name)
+
+	if lint.HasErrors(lintFindings) {
```

Comment:
I understand what you are doing with this. Could we rename this to lint.HasFindings instead to not confuse people in the future?

Reply:
That might be misnomer since we don't want this function to return true if there are findings with a non error severity, maybe hasSevErrorFindings would be better? The goal is to return an error, and ultimately exit 1 the program, only if there are error findings

Reply:
Or simply `HasSeverity(findings, types.SevErr)`